packages/react-i18n/README.md

```diff
@@ -327,30 +327,53 @@ i18n.getTranslationTree('MyComponent.countries');
 
 When rendering internationalized React apps on the server, you will want to extract the translations and rehydrate them on the client if any translations are loaded asynchronously. Not doing so would cause the server and client markup to differ, resulting in a full re-render.
 
-This library uses the [`@shopify/react-effect`](https://github.com/Shopify/quilt/tree/master/packages/react-effect) package to allow translations to be extracted alongside other asynchronous side effects on the server. To make use of this, you will need to keep a reference to the `I18nManager` for your app. Then, import the `extract` function from `@shopify/react-effect`, and call it with your top-level component. Finally, call the manager’s `extract` method to get an opaque representation of the translations that were loaded in that tree:
+This library uses the [`@shopify/react-effect`](https://github.com/Shopify/quilt/tree/master/packages/react-effect) package to allow translations to be extracted alongside other asynchronous side effects on the server. To make use of this, you will need to keep a reference to the `I18nManager` for your app. Then, import the `extract` function from `@shopify/react-effect`, and call it with your top-level component. Call the manager’s `extract` method to get an opaque representation of the translations that were loaded in that tree.
+
+We recommend you to use [`@shopify/react-html`](https://github.com/Shopify/quilt/tree/master/packages/react-html) to serialize the extracted translations.
```

Comment:
actually, `@shopify/react-i18n-universal-provider` does a lot of these now

Reply:
Yes, My first version was using `@shopify/react-i18n-universal-provider` but then I was wondering if we should plug all the other packages into this readme or we should let the readme of `@shopify/react-i18n-universal-provider` explain everything.

Thought?

Reply:
I can't think of any reason why someone would want to set this up themselves instead of using `@shopify/react-i18n-universal-provider`, so lets keep just link out to it.

Reply:
Cool I updated the section and the example to use `@shopify/react-i18n-universal-provider`